core/src/main/java/io/grpc/ServerImpl.java

```diff
@@ -249,6 +257,7 @@ public void transportTerminated() {
     @Override
     public ServerStreamListener streamCreated(final ServerStream stream, final String methodName,
         final Metadata.Headers headers) {
+      final Future<?> timeoutFuture = scheduleTimeout(stream, headers);
```

Comment:
A transport.ServerStream is a Call no?  Each "RPC" on the wire is a separate HTTP2 Stream, which is what a ServerStream is I thought.